swatch-core/src/main/java/org/candlepin/subscriptions/db/model/SubscriptionCapacityView.java

```diff
@@ -104,8 +104,8 @@
   @Column(name = "account_number")
   private String accountNumber;
 
-  @Column(name = "has_unlimited_guest_sockets")
-  private boolean hasUnlimitedGuestSockets;
+  @Column(name = "has_unlimited_usage")
+  private boolean hasUnlimitedUsage;
```

Comment:
This should be Boolean as well to avoid this issue:

```sql
insert into subscription_capacity(account_number, product_id, subscription_id, owner_id, physical_sockets, virtual_sockets, has_unlimited_usage, begin_date, end_date, physical_cores, virtual_cores, sku, sla, usage) values ('account123', 'RHEL', 'subscription123', 'org123', 1, 2, null, '2021-01-01', '2023-01-01', 1, 1, 'SKU', 'Premium', 'Production')
insert into subscription(sku, owner_id, subscription_id, quantity, start_date, end_date, marketplace_subscription_id, account_number, subscription_number) values ('SKU', 'org123', 'subscription123', 1, '2021-01-01', '2023-01-01', null, 'account123', 'subscription123');
insert into offering(sku, product_name, product_family, physical_cores, physical_sockets, virtual_cores, virtual_sockets, role, sla, usage, description, has_unlimited_usage) values ('SKU', 'RHEL dude', 'RHEL', 1, 1, 1, 1, null, null, null, null, null);
```

````
curl -X 'GET' \
  'http://localhost:8000/api/rhsm-subscriptions/v1/subscriptions/products/RHEL' \
  -H 'accept: application/vnd.api+json' \
  -H 'x-rh-identity: eyJpZGVudGl0eSI6eyJhY2NvdW50X251bWJlciI6ImFjY291bnQxMjMiLCJ0eXBlIjoiVXNlciIsInVzZXIiOnsiaXNfb3JnX2FkbWluIjp0cnVlfSwiaW50ZXJuYWwiOnsib3JnX2lkIjoib3JnMTIzIn19fQo='